### PR TITLE
FortyTwo Code of Conduct

### PR DESCRIPTION
**Pull request by andrewconner**

### CODE REVIEW

**Comment by eishay** [comment]
cool

File: kifi-backend/modules/common/public/conduct.html [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+      ​If you think that you are, or any of your colleagues are not following
+      this Code of Conduct or FortyTwo’s Privacy Policy, please don’t be quiet
+      - speak, report and prevent any behaviour that might compromise Kifi’s
+      integrity.
+
+
+    <li>
+      <p><strong>Limit Your Access to Data</strong><br>
+      Our users are keeping different pages that they find interesting and
+      important. They are also convesing using Kifi on private issues. Looking
```
Comment (2is10) [inline kifi-backend/modules/common/public/conduct.html:93]: What does this line mean?

File: kifi-backend/modules/common/public/conduct.html [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+      and searching on a user’s kifi data history can tell us a lot about that
+      user, but sometime it can tell us a lot more that they intended for us to
```
Comment (2is10) [inline kifi-backend/modules/common/public/conduct.html:95]: sometime → sometimes
that → than

File: kifi-backend/modules/common/public/conduct.html [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+      know. Therefore, even when you are needed to access a specific Kifi
+      user’s data, please respect our users privacy and do not access any data
```
Comment (2is10) [inline kifi-backend/modules/common/public/conduct.html:97]: users → users’

File: kifi-backend/modules/common/public/conduct.html [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+      that is not necessary for your work. To be clear, you may not query or
+      access a specific user’s data unless it’s:
+
+      <ol type="A">
+        <li>To technically investigate or fix a problem related to a suspected
+        malfunction with Kifi’s functionality.
+
+        <li>To provide support for this user when the user reports a problem or
+        a question..
```
Comment (2is10) [inline kifi-backend/modules/common/public/conduct.html:106]: .. → .

File: kifi-backend/modules/common/public/conduct.html [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+
+        <li>To understand behavior when planning or building a feature, not
+        targeting specific user’s personal information.
```
Comment (2is10) [inline kifi-backend/modules/common/public/conduct.html:109]: a specific user’s  or   specific users’

File: kifi-backend/modules/common/public/conduct.html [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+      ― <cite>Douglas Adams, The Hitchhiker’s Guide to the Galaxy</cite>
+    </footer>
+  </blockquote>
+
+  <p>If you have any doubts on whether your actions might breach the Code of
+  Conduct, feel free to review the Code of Conduct and Privacy Policy and don’t
+  hesitate to ask questions. We want you not only to read the code, but to
+  actually understand it and help us improve it.
+
+  <h3>We expect all 42 employees to not only read the code, but to follow and
```
Comment (2is10) [inline kifi-backend/modules/common/public/conduct.html:147]: FortyTwo

**Comment by 2is10** [comment]
:+1:

**Pull request merged**